tsl/test/expected/compression_ddl.out

```diff
@@ -54,6 +54,48 @@ DROP INDEX new_index;
 ALTER TABLE test1 SET (fillfactor=100);
 ALTER TABLE test1 RESET (fillfactor);
 ALTER TABLE test1 ALTER COLUMN b SET STATISTICS 10;
+-- ensure that REPLICA IDENTITY works
+ALTER TABLE test1 REPLICA IDENTITY FULL;
+SELECT relname, relreplident FROM pg_class WHERE relname = 'test1' ORDER BY relname;
+ relname | relreplident 
+---------+--------------
+ test1   | f
+(1 row)
+
+-- the chunk's setting should also change to FULL
+SELECT relname, relreplident FROM show_chunks('test1') ch INNER JOIN pg_class c ON (ch = c.oid) ORDER BY relname;
```

Comment:
```suggestion
SELECT relreplident, COUNT(*) FROM show_chunks('test1') ch INNER JOIN pg_class c ON (ch = c.oid) GROUP BY 1 ORDER BY 1;
```
Decrease the test output size... also be good to run it before and after change the replica identity.

Reply:
done